src/utils/formatNumber.ts

```diff
@@ -57,31 +58,29 @@ const getCurrencyFormatterOptions = (float: number, currency: string): Intl.Numb
   }
 }
 
-// TODO: Add signs in reference to final point of number formatting
-const formatNumber = (float: number, options: Intl.NumberFormatOptions): string => {
-  const VISIBLE_UPPER_LIMIT = 999 * 10 ** 12 // 999T
-
+const formatNumber = (float: number, options: Intl.NumberFormatOptions, keepSign?: boolean): string => {
```

Comment:
We can determine whether the "Eucledian distance of the amount is smaller than 0.00001" without the `keepSign` argument. I don't think it's necessary as we format numbers the same app wide.

Reply:
I removed the argument and preferred to check if the string starts with the plus sign